tsconfig.json

```
@@ -1,7 +1,30 @@
 {
-    "files": [],
-    "references": [
-        { "path": "./tsconfig.app.json" },
-        { "path": "./tsconfig.node.json" }
-    ]
+    "compilerOptions": {
+        "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.app.tsbuildinfo",
+        "target": "ES2023",
+        "useDefineForClassFields": true,
+        "lib": ["ES2023", "DOM", "DOM.Iterable"],
+        "module": "ESNext",
+        "skipLibCheck": true,
+
+        /* Bundler mode */
+        "moduleResolution": "bundler",
+        "allowImportingTsExtensions": true,
+        "verbatimModuleSyntax": true,
+        "moduleDetection": "force",
+        "noEmit": true,
+        "allowJs": true,
+        "jsx": "react-jsx",
+
+        /* Linting */
+        "strict": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "erasableSyntaxOnly": true,
+        "noFallthroughCasesInSwitch": true,
+        "noUncheckedSideEffectImports": true,
+        "noImplicitAny": true
+    },
+    "include": ["vite.config.ts", "src/api", "src/common", "src/app"],
+    "exclude": ["test-utils.tsx"]
```

Otherwise "build" directories are checked and test-utils.tsx is already fixed so we can just include it.

```suggestion
    "include": ["vite.config.ts", "test-utils.tsx", "src/api", "src/common", "src/app"],
    "exclude": ["src/build", "src/app/build"]
```